### ISSUE TITLE
Last fixes and touches for gradian descent & logistic Regression

### ISSUE THREAD

**Issue by AseelAborokn** [sirandreww/ml_tutor_frontend#16]
| Gradient Descent |
| ------------- |
| bug - when we start typing `e` in the function box, the site crashes |
| Add a new page that describes which mathematical functions we can write (examples like how to write log/ln/exponent...) |
| if you have a nice 1D/2D function that represents local minimal add these cases as descriptions in the playgrounds |

| Logistic Regression |
| ------------- |
| Our dataset is very good thus last task (importance of hyper-parameters) is not needed, if you can find larger dataset or another dataset which has problems it will be great |
